vsintegration/src/vs/FsPkgs/FSharp.Project/FS/Project.fs

```diff
@@ -724,7 +721,6 @@ See also ...\SetupAuthoring\FSharp\Registry\FSProjSys_Registration.wxs, e.g.
                 this.AddCATIDMapping(typeof<OAFSharpFileItem>, typeof<OAFSharpFileItem>.GUID)
 #endif
                 // The following are not specific to F# and as such we need a separate GUID (we simply used guidgen.exe to create new guids)
```

Comment:
If we are deleting the next line perhaps this comment should go to, since I think it refers to the deleted line.